From 9606bcbadf11728b270011ca53699a4e8dfc27c0 Mon Sep 17 00:00:00 2001
From: Alan Pevec <apevec@redhat.com>
Date: Thu, 8 Nov 2012 22:58:54 +0100
Subject: [PATCH] Fix default port for identity.internalURL

This should be the public_port and not the admin one.

Sync with default_catalog.templates change
commit 773f0f84af282cd3e53650ccbb99284c37677b6a

Change-Id: Ibb81a06607d94648993768c8b2e7161bf57195a1
---
 tools/sample_data.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/sample_data.sh b/tools/sample_data.sh
index ced89dc..a37a640 100755
--- a/tools/sample_data.sh
+++ b/tools/sample_data.sh
@@ -188,7 +188,7 @@ if [[ -n "$ENABLE_ENDPOINTS" ]]; then
     keystone endpoint-create --region RegionOne --service_id $KEYSTONE_SERVICE \
         --publicurl 'http://localhost:$(public_port)s/v2.0' \
         --adminurl 'http://localhost:$(admin_port)s/v2.0' \
-        --internalurl 'http://localhost:$(admin_port)s/v2.0'
+        --internalurl 'http://localhost:$(public_port)s/v2.0'
 fi
 
 VOLUME_SERVICE=$(get_id \
